xclim/indices/_agro.py

```diff
@@ -341,3 +344,71 @@ def water_budget(
 
     out.attrs["units"] = pr.attrs["units"]
     return out
+
+
+@declare_units(pr="[precipitation]", thresh="[precipitation]")
+def rolling_drydays_events(
```

Comment:
What about `dry_spells_frequency` and `dry_spells_total_length` ?
To mirror the ``hot_spells`, `heat_wave`, etc,  names. @tlogan2000

Reply:
> 
> 
> What about `dry_spells_frequency` and `dry_spells_total_length` ?
> To mirror the `hot_spells`, `heat_wave`, etc, names. @tlogan2000

Ok for me. I think I would say `dry_spell_frequency` and `dry_spell_total_length`...   i,e, not "spells"